### ISSUE TITLE
Windows

### ISSUE THREAD

**Issue by gedw99** [mitchellh/libxev#66]
Maybe in the future ? 

Bun also can’t fully work on windows

**Comment by mitchellh** [comment]
Initial support #42

**Comment by rsepassi** [comment]
Don't have a clean repro at the moment, but initial check of the Windows support indicated that 2 timers were running sequentially instead of in parallel; could very well have been holding it wrong, but the same code worked fine on Linux and Mac. cc @Corendos

**Comment by Corendos** [comment]
Hey @rsepassi, could you open a new issue so that we can discuss this ? :slightly_smiling_face:
